src/gdcdictionary/schemas/follow_up.yaml

```diff
@@ -1894,6 +1894,7 @@ properties:
     items:
       enum:
         - Abdomen
+        - Cervix Uteri
```

Comment:
the capitalization does not match the ticket.

Reply:
Fixed in JIRA ticket, should be title case